### PR TITLE
Prepare for release 2.3.1

### PR DESCRIPTION
**Pull request by ilyasfoo**
### Changes proposed in this Pull Request:

Preparation for release of 2.3.1

### Changelog

```
= 2.3.1 =
* Hide free trial plan picker banner when viewing iframe #1404
* Force remove "Need help?" spotlight in tasklist #1417

```

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -22,7 +22,7 @@ This section describes how to install the plugin and get it working.
 
 == Changelog ==
 
-= Unreleased
+= 2.3.1 =
 * Hide free trial plan picker banner when viewing iframe #1404
```
Comment (chihsuan) [inline readme.txt:26]: This is already included in 2.3.0. 😬

Comment (ilyasfoo) [reply readme.txt:26]: Ah, I see. Will fix that

**Review by chihsuan — APPROVED** [review]
LGTM 🚀

File: readme.txt (hunk 1 above)
Comment (ilyasfoo) [reply readme.txt]: @chihsuan Fixed it in [61215c8](https://github.com/Automattic/wc-calypso-bridge/pull/1418/commits/61215c845a00904e553798f2edecf6b41f2725fe), there was copy pasta issue

**Pull request merged**